server/api/api/endpoints/model_endpoints.py

```diff
@@ -14,6 +14,7 @@
 
 import asyncio
 import json
+import typing
```

Comment:
It can be nicer to use `from typing import ...` and combine with this line below

Reply:
Alright, but `make fmt` changes that to `from collections.abc import Coroutine`.

Reply:
https://docs.python.org/3/library/typing.html#typing.Coroutine

> Deprecated since version 3.9: [collections.abc.Coroutine](https://docs.python.org/3/library/collections.abc.html#collections.abc.Coroutine) now supports subscripting ([]). See [PEP 585](https://peps.python.org/pep-0585/) and [Generic Alias Type](https://docs.python.org/3/library/stdtypes.html#types-genericalias).

Reply:
Ok, but then I need to change it back, because ruff insists on changing `from typing import Coroutine` to this.

Reply:
```py
from collections.abc import Coroutine
```
Should be fine...
`typing.Coroutine` is deprecated, as I quoted above.